internal/impl/global.go

```diff
@@ -142,7 +142,7 @@ func (d *Devbox) pullGlobalFromPath(path string) error {
 }
 
 func (d *Devbox) addFromPull(pullCfg *Config) error {
-	if pullCfg.Nixpkgs.Commit != plansdk.DefaultNixpkgsCommit {
+	if pullCfg.NixPkgsCommitHash() != plansdk.DefaultNixpkgsCommit {
```

Comment:
1. Why do we still need this change?
2. Do we still require `plansdk.DefaultNixpkgsCommit` or could we get rid of it too?